assets/css/app.css

```diff
@@ -0,0 +1,68 @@
+body {
+  font-family: 'Poiret One', cursive;
+  font-size 1.6em
```

Comment:
Very unusual to set `em`s on the body directly. This is because the initial value for `em` is usually taken *from* the body.

So if the body has `16px` as the font-size, all `em`s are subsequently based off the idea that `1em = 16px` (unless you change font sizes in parent elements)